packages/flutter/test_fixes/cupertino/cupertino.dart.expect

```diff
@@ -246,46 +257,51 @@ void main() {
   );
   final Clip clip = details.decorationClipBehavior;
 
-  final PlatformMenuBar platformMenuBar = PlatformMenuBar(menus: <PlatformMenuItem>[], child: const SizedBox());
+  final PlatformMenuBar platformMenuBar = PlatformMenuBar(
+    menus: <PlatformMenuItem>[],
+    child: const SizedBox(),
+  );
   final Widget bodyValue = platformMenuBar.child;
 
   // Changes made in https://github.com/flutter/flutter/pull/151367
-  final cupertinoSwitch =  CupertinoSwitch(
+  final cupertinoSwitch = CupertinoSwitch(
     value: value,
     onChanged: onChanged,
     activeTrackColor: Colors.red,
   );
-  Color? activeTrackColor =  cupertinoSwitch.activeTrackColor;
+  Color? activeTrackColor = cupertinoSwitch.activeTrackColor;
 
   // Changes made in https://github.com/flutter/flutter/pull/151367
-  final cupertinoSwitch =  CupertinoSwitch(
+  final cupertinoSwitch = CupertinoSwitch(
     value: value,
     onChanged: onChanged,
     inactiveTrackColor: Colors.red,
   );
-  Color? inactiveTrackColor =  cupertinoSwitch.inactiveTrackColor;
+  Color? inactiveTrackColor = cupertinoSwitch.inactiveTrackColor;
 
   // https://github.com/flutter/flutter/pull/152981
   CupertinoCheckbox(fillColor: WidgetStateProperty.resolveWith<Color>((Set<WidgetState> states) {
-    if (states.contains(WidgetState.disabled)) {
-      return CupertinoColors.white.withOpacity(0.5);
-    }
-    if (states.contains(WidgetState.selected)) {
-      return CupertinoDynamicColor.resolve(CupertinoDynamicColor.withBrightness(
-        color: CupertinoColors.activeBlue,
-        darkColor: Color.fromARGB(255, 50, 100, 215),
-      ), context);
-    }
-    return Colors.red;
-  }));
+ if (states.contains(WidgetState.disabled)) {
+ return CupertinoColors.white.withOpacity(0.5);
+ }
+ if (states.contains(WidgetState.selected)) {
+ return CupertinoDynamicColor.resolve(CupertinoDynamicColor.withBrightness(
+ color: CupertinoColors.activeBlue,
+ darkColor: Color.fromARGB(255, 50, 100, 215),
+ ), context);
+ }
+ return Colors.red;
+ }));
   CupertinoCheckbox(activeColor: Colors.white, fillColor: WidgetStateProperty.resolveWith<Color>((Set<WidgetState> states) {
-    if (states.contains(WidgetState.disabled)) {
-      return CupertinoColors.white.withOpacity(0.5);
-    }
-    if (states.contains(WidgetState.selected)) {
-      return Colors.white;
-    }
-    return Colors.red;
-  }));
-  CupertinoCheckbox(fillColor: WidgetStatePropertyAll(CupertinoColors.white));
+ if (states.contains(WidgetState.disabled)) {
+ return CupertinoColors.white.withOpacity(0.5);
```

Comment:
This is an example of dart fix no longer inserting the whitespace it did previously.